### PR TITLE
Fix missing localization of inactive user removal warning emails.

### LINKED ISSUE #5950: account_removal_warning mail not localized into the users language
All mails get send out in English, which is rather unfortunate.

<bountysource-plugin>
---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/14407523-account_removal_warning-mail-not-localized-into-the-users-language?utm_campaign=plugin&utm_content=tracker%2F522&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F522&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by jaywink**
Fixes #5950

### CODE REVIEW

File: app/mailers/maintenance.rb [hunk 1]
```diff
@@ -7,9 +7,12 @@ def account_removal_warning(user)
     @pod_url = AppConfig.environment.url
     @after_days = AppConfig.settings.maintenance.remove_old_users.after_days.to_s
     @remove_after = @user.remove_after
-    mail(:to => @user.email, :subject => I18n.t('notifier.remove_old_user.subject')) do |format|
-      format.text
-      format.html
+
+    I18n.with_locale(@user.language) do
+      mail(:to => @user.email, :subject => I18n.t('notifier.remove_old_user.subject')) do |format|
```
Comment (jhass) [inline app/mailers/maintenance.rb]: Also switch to Ruby 1.9 style hash syntax here while touching it (`foo: bar, baz: quux`).

**Comment by denschub** [comment]
Merged as ce16a122fcb0283f2af47e3e1aa379af4449e591.

Please don't add Changelog entries in the future. The merge conflicts are a bit annoying. :)

**Comment by jaywink** [comment]
@denschub you mean we don't expect pull requests to contain changelogs any more?

**Comment by jhass** [comment]
@jaywink I wrote this little helper: https://gist.github.com/jhass/48c6ec1aa732d7287432 ;)

With that adding the changelog entry is easier done by the merger (= less conflicts).

**Comment by denschub** [comment]
You can add it if you really want to, resolving that conflict is very easy. But you don't have to.

If you want, you could edit the pull requests title to match your changelog entry, so one could easily c&p them.

**Comment by jaywink** [comment]
No I totally agree pull requests shouldn't be assumed to contain changelogs at all, this is a good change imho. We've just always requested them before, that is why I included them.. ;)

**Pull request closed without merging**